app/main/views/frameworks.py

```diff
@@ -122,23 +122,30 @@ def framework_dashboard(framework_slug):
 @main.route('/frameworks/<framework_slug>/signing', methods=['GET'])
 @login_required
 def framework_signing(framework_slug):
+    print('START OF CONTROLLER')
     framework = get_framework(data_api_client, framework_slug, allowed_statuses=("standstill", "live", "expired",))
-
+    print('FRAMEWORK: ')
+    print(framework)
     if not framework.get("frameworkAgreementVersion"):
+        print('FRAMEWORK DOESNT HAS AGREEMENT')
         # presumably an old pre-g8 framework that we don't want to enable this feature for
         abort(404)
 
+    print('FRAMEWORK HAS AGREEMENT')
     supplier_framework_info = get_supplier_framework_info(data_api_client, framework_slug)
+    print('SUPPLIER FRAMEWORK INFO: ')
+    print(supplier_framework_info)
     if not get_supplier_on_framework_from_info(supplier_framework_info):
-        abort(404)
 
-    drafts, complete_drafts = get_drafts(data_api_client, framework_slug)
-    applied_lot_slugs = frozenset(draft["lotSlug"] for draft in complete_drafts)
+        abort(404)
+    print('FRAMEWORK HAS FRAMEWORK INFO')
+    # drafts, complete_drafts = get_drafts(data_api_client, framework_slug)
+    # applied_lot_slugs = frozenset(draft["lotSlug"] for draft in complete_drafts)
 
+    print('RENDERING TEMPLATE')
```

Comment:
Hold on @laurenceberry - did you mean to include all these changes to the view? What's the intent here? (apart from the debugging lines)